requirements.txt

```diff
@@ -49,41 +45,38 @@ django-redis==4.8.0
 # Celery support (needed for Thorn)
 celery<=4.2
 
-thorn==1.5.0
-
-# Wharton styles
-git+https://github.com/jefftriplett/django-base-theme.git@432f87f  # Pinned to the latest
-django-bootstrap3==8.2.1
+# thorn==1.5.2
+git+https://github.com/simplworld/thorn.git@upgrade-to-django-2-dot-2  # Our fork until the main project is upgraded to Django 2.2
```

Comment:
This repo's README states:
```
Thorn version 1.0 runs on
    Python (2.7, 3.4, 3.5)
    PyPy (5.1.1)
    Jython (2.7).
    Django (1.8, 1.9, 1.10)
```
Is that accurate?

Reply:
Our upstream pull request was accepted. 🎉  https://github.com/robinhood/thorn/pull/38

So I think I can update and remove our forked copy which will resolve this. I'll update after my meetings this morning.

Reply:
The Thorn project accepted our PR so I deleted our fork which resolves this.